submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post2.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post3.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post4.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+      </div>
+    </main>
+  </body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

Reply:
Fixed in the last commit